sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts

```diff
@@ -3,225 +3,41 @@
 
 /* eslint-disable @typescript-eslint/no-unused-vars */
 
-import { TokenCredential, GetTokenOptions, AccessToken } from "@azure/core-http";
+import { credentialLogger } from "../util/logging";
+import { MsalOpenBrowser } from "../msal/nodeFlows/msalOpenBrowser";
 import { InteractiveBrowserCredentialOptions } from "./interactiveBrowserCredentialOptions";
-import { credentialLogger, formatError, formatSuccess } from "../util/logging";
-import { DefaultTenantId, DeveloperSignOnClientId } from "../constants";
-import { Socket } from "net";
-import { AuthenticationRequired, MsalClient } from "../client/msalClient";
-import { AuthorizationCodeRequest } from "@azure/msal-node";
-
-import open from "open";
-import http from "http";
-import stoppable from "stoppable";
-
-import { checkTenantId } from "../util/checkTenantId";
+import { InteractiveCredential } from "./interactiveCredential";
 
 const logger = credentialLogger("InteractiveBrowserCredential");
 
 /**
  * Enables authentication to Azure Active Directory inside of the web browser
- * using the interactive login flow, either via browser redirects or a popup
- * window.
+ * using the interactive login flow.
+ *
+ * On NodeJS, it will open a browser window while it listens for a redirect response from the authentication service.
+ *
+ * On browsers, by default it uses the [Authorization Code Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow)
+ * and authenticates via popups. The `loginStyle` can be configured to use `redirect` instead, and the authentication flow can be configured to use the [Implicit Grant Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow)
```

Comment:
You reference `loginStyle` as though it's an option, but the option appears to be `flow`.

Reply:
loginStyle is for popup and redirect, and flow is for auth-code or implicit. Both are options.

Reply:
I see it was added before this PR. However, it seems it's only applicable to `InteractiveBrowserCredentialBrowserOptions` (which makes sense since node would essentially always be pop-up), so perhaps we should remove it from the doc comments for the node implementation?

Reply:
Ahh thanks! I'll fix it.